Dockerfile

```diff
@@ -28,6 +28,10 @@ COPY --from=build /tacacs /tacacs
 COPY tac_plus.sample.cfg /etc/tac_plus/tac_plus.cfg
 COPY docker-entrypoint.sh /docker-entrypoint.sh
 
+
+RUN apt update && \
+    apt install -y --no-install-recommends libdigest-md5-perl libnet-ldap-perl
+
```

Comment:
Can you please add `rm -rf /var/cache/apk/*` to the end of the run block? This will remove the apt cache from the final image.